shell/components/form/ResourceTabs/index.vue

```diff
@@ -75,23 +75,25 @@ export default {
     const inStore = this.$store.getters['currentStore'](EVENT);
     const eventSchema = this.$store.getters[`${ inStore }/schemaFor`](EVENT); // @TODO be smarter about which resources actually ever have events
 
+    const paginationHeaders = eventSchema ? [
```

Comment:
`paginationHeaders` can be an empty array since the UI shows events only if event schema is not null:

```
    showEvents() {
      return this.isView && this.needEvents && this.eventSchema;
    },
```